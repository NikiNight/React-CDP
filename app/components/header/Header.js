import React from 'react';
import MovieFounded from './MovieFounded';
import MovieSearch from './MovieSearch';
import HeaderLine from './HeaderLine';
 
const Header = (props) => {

    return (
        <header className="header">
            <div className="wrapper">
                <HeaderLine 
                    isMovieSelected={props.isMovieSelected}
                    backToSearch={props.backToSearch}
                />
                {
                    props.isMovieSelected ?
                    <MovieFounded selectedMovie={props.selectedMovie} /> :
                    <MovieSearch/>
                }
            </div>
        </header>
    )
}

export default Header;