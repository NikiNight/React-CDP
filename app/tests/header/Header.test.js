import React from 'react';
import renderer from 'react-test-renderer';
import Header from './../../components/header/Header';

test('matches snapshot', () => {
  const tree = renderer.create(<Header />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});