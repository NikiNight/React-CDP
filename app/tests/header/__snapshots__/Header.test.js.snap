// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<header
  className="header"
>
  <div
    className="wrapper"
  >
    <div
      className="header-line"
    >
      <a
        className="logo"
        href="#"
      >
        netflixroulette
      </a>
    </div>
    <div
      className="movie-search"
    >
      <h1>
        Find your movie
      </h1>
      <input
        className="movie-search__input"
        name="movie"
        type="text"
      />
      <div
        className="movie-search__buttons-line"
      >
        <div
          className="movie-search__buttons"
        >
          <span>
            Search by
          </span>
          <button
            className="button button_active"
            onClick={[Function]}
            value="Title"
          >
            Title
          </button>
          <button
            className="button "
            onClick={[Function]}
            value="Genre"
          >
            Genre
          </button>
        </div>
        <button
          className="button button_search"
          onClick={[Function]}
        >
          Search
        </button>
      </div>
    </div>
  </div>
</header>
`;
